morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala

```diff
@@ -1,7 +1,9 @@
 package org.finos.morphir.runtime
 import org.finos.morphir.naming.*
 
+private[runtime] case class CoverageInfo(staticallyReachedFunctions: List[FQName], userDefinedFunctions: List[FQName], counts: CoverageCounts)
 case class TestSummary(
+    private val coverage: CoverageInfo,
```

Comment:
Failing to understand the access restrictions here. What purpose are they to serve?

Reply:
Also why are we breaking a contract by adding an item to the front?

Reply:
The private restriction is there because we expect to provide more detailed coverage info in the future, and didn't want to break things when they're added.

Reply:
You're right, it would be better to add CoverageInfo to the back with a default value... honestly I should have made the whole `TestSummary` case class private in the first place, there's too much additional information we're likely to attach to that.

Reply:
- That was my bad adding to the front and I have moved it. 
- The thinking on the access restriction is this is likely to change so we didn't wan't users to rely on it

Reply:
Exactly. Hate to sound like a broken record but once we cut a release we have to think about the impact of breaking changes. Especially once production systems depend on things.

Not properly managing breaking changes is asking for bringing down systems unexpectedly